helm_deploy/values-prod.yaml

```diff
@@ -25,23 +39,9 @@ generic-service:
   petty-france-wifi: "213.121.161.112/28"
   global-protect: "35.176.93.186/32"
 
-  postgresDatabaseRestore:
```

Comment:
I think accidentally having this indented to be underneath `allowList` caused the ingress to not be happy, see [Slack message](https://mojdt.slack.com/archives/CP8FR9XL5/p1680256290698449?thread_ts=1680256172.120439&cid=CP8FR9XL5) for alerts